aequilibrae/paths/linear_approximation.py

```diff
@@ -0,0 +1,308 @@
+import numpy as np
+from scipy.optimize import root_scalar
+from typing import List
+
+from aequilibrae.paths.traffic_class import TrafficClass
+from aequilibrae.paths.all_or_nothing import allOrNothing
+from aequilibrae import Parameters
+from aequilibrae import logger
+
+if False:
+    from aequilibrae.paths.traffic_assignment import TrafficAssignment
+
+
+class LinearApproximation:
+    def __init__(self, assig_spec, algorithm) -> None:
+        self.algorithm = algorithm
+        parameters = Parameters().parameters["assignment"]["equilibrium"]
```

Comment:
Is this a global variable?

Reply:
There is a parameter file that ships with AequilibraE.
Documentation still incomplete:  https://aequilibrae.com/python/dev/parameter_file.html
But there is a little example on how to manipulate it: https://aequilibrae.com/python/dev/usageexamples.html#parameters-module